src/Plugins/DavisIMipPlugin.php

```diff
@@ -75,6 +75,15 @@ public function schedule(ITip\Message $itip)
         $senderEmail = substr($itip->sender, 7);
         $recipientEmail = substr($itip->recipient, 7);
 
+        if ($itip->senderName) {
+            $senderName = $itip->senderName.' <'.$senderEmail.'>';
+        } else {
+            $senderName = $senderEmail;
+        }
+        if ($itip->recipientName && $itip->recipientName != $recipientEmail) {
+            $recipientName = $itip->recipientName.' <'.$recipientEmail.'>';
```

Comment:
Same as my comment above

Reply:
This is ultimately the same as above so I will mark this as resolved pending the above conversation